mumax3c/scripts/energy.py

```diff
@@ -73,40 +87,44 @@ def dmi_script(system):
     if mm.Exchange() not in system.energy:
```

Comment:
I think it might be worth giving an error here rather than setting a very small exchange.

Reply:
I think so too. Should we give an error or a warning?

Reply:
I think an error.

Reply:
I think an error would be good.